spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -1181,6 +1218,113 @@ else if (handler instanceof HandlerCollection) {
 			});
 		}
 
+		public static class Accesslog {
+
+			/**
+			 * Enable access log.
+			 */
+			private Boolean enabled;
+
+			/**
+			 * Jetty accesslog filename.
+			 */
+			private String filename;
+
+			/**
+			 * the number of days before rotated log files are deleted.
+			 */
+			private int retainDays;
+
+			/**
+			 * append to log flag.
+			 */
+			private boolean append;
+
+			/**
+			 * the log file name date format.
+			 */
+			private String filenameDateFormat;
+
+			/**
+			 * extended NCSA format.
+			 */
+			private boolean extended;
+
+			/**
+			 * Set the timezone of the request log.
+			 */
+			private String logTimeZone;
+
+			/**
+			 * Controls logging of the request cookies.
+			 */
+			private boolean logCookies;
```

Comment:
Looking at the implementation there are a bunch of items you can log. You've chosen to include cookies only. why? 

Perhaps this can be extended as

```
accesslog.log.cookies=false
accesslog.log.server=false
accesslog.log.latency=false
```